.github/workflows/enos-run.yml

```diff
@@ -107,9 +113,16 @@ jobs:
           ENOS_VAR_terraform_plugin_cache_dir: ./support/terraform-plugin-cache
           ENOS_VAR_vault_bundle_path: ./support/vault.zip
         run: |
-          enos scenario destroy --timeout 60m0s --chdir ./enos ${{ matrix.scenario }} arch:amd64 builder:crt edition:oss
-      - name: Output debug information on failure
-        if: ${{ failure() }}
+          enos scenario destroy --timeout 60m0s --chdir ./enos ${{ matrix.scenario }}
+      # Temporarily Disable debug output
```

Comment:
I would rather either remove this entirely or re-enable it.

Reply:
agree 👍